Integrations/ProofpointTAP_v2/ProofpointTAP_v2.py

```diff
@@ -289,6 +308,14 @@ def main():
             results = test_module(client, fetch_time, event_type_filter)
             return_outputs(results, None)
 
+            
+        elif demisto.command() == 'proofpoint-get-forensics':
+            threat_id=demisto.args().get("threat_id")
+            results = get_forensics(client, threat_id,server_url,api_version,verify_certificate,service_principal,secret)
+            fileres = fileResult('Forensics_Report.txt',json.dumps(results,indent=1))
+            demisto.results(fileres)
+            return_outputs({'Forensics':fileres},{'Forensics':fileres},None)
```

Comment:
The first parameter of `return_outputs` should be markdown, please convert `fileres` into markdown. You can use `tableToMarkdown` function.
https://github.com/demisto/content/tree/master/docs/code_conventions#tabletomarkdown